servers/src/test/java/tachyon/worker/block/evictor/EvictorTest.java

```diff
@@ -66,16 +73,14 @@
   public static Collection<Object[]> data() {
     // Run this test against all types of Evictors
     List<Object[]> list = new ArrayList<Object[]>();
-    for (EvictorType type : EvictorType.values()) {
-      if (type != EvictorType.DEFAULT) {
-        list.add(new Object[] {type});
-      }
-    }
+    list.add(new Object[] {EvictorConstants.GREEDY_EVICTOR_CLASS.getName()});
+    list.add(new Object[] {EvictorConstants.LRU_EVICTOR_CLASS.getName()});
+
```

Comment:
Each new evictor needs to add its own class name here for evictor contract test now, before change, they only need to be defined in the `EvictorType`. 

What about having a `public static final Class<? extends Evictor>[] evictorTypes = new xxx { GREEDY_EVICTOR_CLASS, LRU_EVICTOR_CLASS }; ` in `EvictorConstants`, so that when a new evictor is added, we add its type to `EvictorConstants` and `evictorTypes`, and in `data()`, we can use `evictorTypes` ?